src/date-picker/date-picker.jsx

```diff
@@ -149,11 +159,11 @@ const DatePicker = React.createClass({
 
   getDefaultProps() {
     return {
-      formatDate: DateTime.format,
+      formatDate: toIsoDateString,
       autoOk: false,
       disableYearSelection: false,
       style: {},
-      firstDayOfWeek: 0,
+      firstDayOfWeek: 1,
```

Comment:
P.S this is a breaking change. I think we should add this to the head so we won't forget. ( sorry for nitpicking )